### PR TITLE
feature: make it possible to add tooltips.

### PR DESCRIPTION
**Pull request by aleksei-korolev**
### Description

I have implemented tooltips by accident while working on TRNG-1005. No clue how did I end up doing it.

Add the [Tooltip(string)] attribute (not UnityEngine.TooltipAttribute) to any property or field in your IData. Hover over its label in the step inspector. Here's your tooltip.

### Type of change

- [X] New feature (non-breaking change which adds functionality)

### How Has This Been Tested?

I tested it with properties inside behavior's data, and inside the step data. It worked.

### Checklist

- [X] My code follows the [Coding Conventions](#coding-conventions)
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by aleksei-korolev** [comment]
This ticket is not part of the current sprint. Put the code review at the lowest priority.

File: Editor/UI/Windows/StepWindow.cs [hunk 1]
```diff
@@ -72,7 +72,7 @@ private void OnGUI()
             scrollPosition = GUI.BeginScrollView(new Rect(0, 0, position.width, position.height), scrollPosition, stepRect, false, false);
             {
                 Rect stepDrawingRect = new Rect(stepRect.position + new Vector2(border, border), stepRect.size - new Vector2(border * 2f, border * 2f));
-                stepDrawingRect = drawer.Draw(stepDrawingRect, step, ModifyStep, "Step");
+                stepDrawingRect = drawer.Draw(stepDrawingRect, step, ModifyStep, new GUIContent("Step"));
```
Comment (Gusinuhe) [inline Editor/UI/Windows/StepWindow.cs:75]: Can this be cached somehow?

File: Runtime/Attributes/TooltipAttribute.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+using System.Reflection;
+
+namespace Innoactive.Creator.Core.Attributes
+{
+    public class TooltipAttribute : MetadataAttribute
```
Comment (Gusinuhe) [inline Runtime/Attributes/TooltipAttribute.cs:5]: Missing docs in the whole class

**Comment by Luphoxa** [comment]
there is a ticket that wants the possibility of tool tips: https://jira.innoactive.de/browse/TRNG-410
is this related?

**Comment by Gusinuhe** [comment]
What is this PR's status? Is it abandoned? Can we continue with it?

**Comment by Luphoxa** [comment]
I would like to know as well :) Maybe we should groom the related ticket https://jira.innoactive.de/browse/TRNG-410 in one of the next groomings, so we know what the team thinks about this :)

**Comment by aleksei-korolev** [comment]
I'd say lets get back to it once we finish with the rest of the sprint.

**Comment by tomwim** [comment]
What do we want to do with this PR? It's from September and we never used it, right? @SimonTheSourcerer @Gusinuhe

**Pull request closed without merging**